vmslave-config/ansible/roles/tmp-clean/files/clear-ci-log

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+year_string=`date +%Y`
+year_log_pattern='d'${year_string}*
+find /tmp -mtime +1 -name ${year_log_pattern}
+find /tmp -mtime +1 -name npm* -execdir rm -rf "{}" \;
+find /tmp -atime +1 -name vagrant* -exec rm -rf "{}" \;
```

Comment:
Does the PR only clear directories and files whose name starts with "npm" or "vagrant"?
I just find there are also many directories whose name starts with "d", such as: d20170628-12047-1gp2yx4
Could you please also clear them?